spec/dummy/config/application.rb

```diff
@@ -17,8 +17,5 @@ class Application < Rails::Application
     # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
-
-    # Do not swallow errors in after_commit/after_rollback callbacks.
-    config.active_record.raise_in_transactional_callbacks = true
```

Comment:
This config is [deprecated](http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#error-handling-in-transaction-callbacks) in Rails 5.0. That's why I decided to update the dependency to >= 5.0 because I think if someone uses the gem with Rails 5 and this config is here it will fail, and the same if someone is using Rails 4 and the config is not there. But since it's in the dummy app I'm not sure. 

I can generate 2 local versions of this gem (with the 2 different dependencies) and test with 2 apps, one in Rails 5 and one in Rails 4, and see if they work.

What do you think?